.github/workflows/rubyonrails.yml

```diff
@@ -0,0 +1,42 @@
+name: "Ruby on Rails CI"
+on:
+  push:
+    branches: [ master ]
```

Comment:
Add in `aws-eb-test` and `aws-eb-master` so that we trigger a build when we push to these branches ahead of a deploy.

Reply:
:+1: